pkg/services/store/object/dummy/dummy_server.go

```diff
@@ -109,10 +111,21 @@ func (i dummyObjectServer) Read(ctx context.Context, r *object.ReadObjectRequest
 		}, nil
 	}
 
-	return &object.ReadObjectResponse{
-		Object:      objVersion,
-		SummaryJson: nil,
-	}, nil
+	rsp := &object.ReadObjectResponse{
+		Object: objVersion,
+	}
+	if r.WithSummary {
+		summary, _, e2 := object.GetSafeSaveObject(&object.WriteObjectRequest{
```

Comment:
ha - this is interesting, so if its not the latest version we will have to recompute the summary on the fly

Reply:
in the dummy implementation... we never actually save the summary (we could)

Also note that by definition, the summary should always be able to be calculated from the raw bytes.  If they are out of sync then something went wrong and the new calculation is correct.